src/test/java/org/jahia/test/unomiapi/glue/EventSteps.java

```diff
@@ -111,4 +113,20 @@ else if (eventParams.get("eventType").equals("updateProperties")) {
 
     }
 
+    @Given("The number of view is 1 for page {string} by API with credentials {string} {string}")
```

Comment:
Can you please have only one entry for 1 or 0 page view? Use the number as a parameter